rkyv/src/util/aligned_vec.rs

```diff
@@ -660,3 +623,139 @@ unsafe impl Send for AlignedVec {}
 unsafe impl Sync for AlignedVec {}
 
 impl Unpin for AlignedVec {}
+
+impl ArchivedAlignedVec {
+    /// Returns the number of elements in the archived vec.
+    #[inline]
+    pub fn len(&self) -> usize {
+        from_archived!(self.len) as usize
```

Comment:
TBH I don't really understand what this is doing. I copied it from `ArchivedVec`.

Reply:
It's a nasty workaround since sometimes the archived type of `self.len` will be a plain `u32` and other times it will be a `rend` type. It's used correctly here.